source/contracts/trading/Order.sol

```diff
@@ -22,7 +22,7 @@ library Order {
 
     uint256 constant MIN_ORDER_VALUE = 10**14;
 
-    enum TradeTypes {
+    enum OrderTypes {
```

Comment:
Consider `Types` here since it will always be called as `Order.<name>` and having `Order` twice is redundant.

We use `orderType` as the variable name because `type` is a reserved word, not because it is a good name.  :)

Reply:
Ah yea, will rename the enum